go.mod

```diff
@@ -14,14 +14,11 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pierrec/lz4 v2.3.0+incompatible // indirect
 	github.com/robfig/cron v1.2.0
-	github.com/terraform-providers/terraform-provider-aws v2.32.0+incompatible
```

Comment:
These providers are needed for the tests to work. 
Most likely why you are having issues with tests.

Reply:
Thanks. I put these back and used the pseudo-version to point to the same commit as before.